drivers/i2c/i2c_ll_stm32.h

```diff
@@ -97,4 +98,11 @@ int i2c_stm32_target_register(const struct device *dev, struct i2c_target_config
 int i2c_stm32_target_unregister(const struct device *dev, struct i2c_target_config *config);
 #endif
 
+/* Get the I2C timing register to be reported by the i2c_stm32_get_config */
+#ifdef CONFIG_I2C_STM32_V1
+#define LL_I2C_GET_TIMING(instance) LL_I2C_ReadReg(instance, CCR)
+#else
+#define LL_I2C_GET_TIMING(instance) LL_I2C_ReadReg(instance, TIMINGR)
+#endif
```

Comment:
To remain consistent with current driver implementation, it would be required to use functions

Reply:
do you mean 
```
#ifdef CONFIG_I2C_STM32_V1
	cfg_timings->timing_setting = LL_I2C_ReadReg(instance, CCR)
#else
	cfg_timings->timing_setting = LL_I2C_ReadReg(instance, TIMINGR)
#endif

```

Reply:
I mean proposing a function `stm32_i2c_get_timing` which will be implemented in `drivers/i2c/i2c_ll_stm32_v1.c` and `drivers/i2c/i2c_ll_stm32_v2.c` files

Reply:
Version with `struct i2c_config_timing current_timing`  to  store the current timing structure set by runtime config

Reply:
Get_config : refer to https://github.com/zephyrproject-rtos/zephyr/pull/64422